frontend/stmt_inferrer.py

```diff
@@ -321,12 +321,19 @@ def _infer_func_def(node, context, solver):
     """Infer the type for a function definition"""
```

Comment:
The thing is that a lot of stubs depend of generic type vars (including the `abs` function), so our problem is reduced to encoding an annotation that uses generic type var, and using it later for the inference. I can work on finding a solution to this shortly.